test/add_model_regression_data.py

```diff
@@ -56,7 +66,7 @@
 
 original_regression_data = "model_regression_data.h5"
 extra_regression_data    = "model_regression_data_new.h5"
-new_model = "NRHybSur3dq8_CCE" # name of new model with regression data in fp_new but not fp_old
+new_model = "SEOBNRv4PHMSur" # name of new model with regression data in fp_new but not fp_old
 
```

Comment:
Didn't look at the code, but is ok that new_model no longer checks for NRHybSur3dq8_CCE?

Reply:
So this script (add_model_regresion.py) is only used to add new model regression data to the model regression hdf5 file. The "new_model" variable name is just pointing to the model whose data was most recently added.

Adding data is "staking"... the hdf5 file is extended to include the new model regression data. 

In short, all of the models (including NRHybSur3dq8_CCE) are still tested as before.